db/migrate/20150304152407_set_widget_defaults.rb

```diff
@@ -0,0 +1,12 @@
+class SetWidgetDefaults < ActiveRecord::Migration
+  def up
+    change_column :widgets, :sizex, :integer, :default => 4
```

Comment:
No need for an aditional migration, you can add the defaults on the one you create the table